webapps-backup/src/main/java/io/camunda/webapps/backup/BackupServiceImpl.java

```diff
@@ -38,14 +38,14 @@ public BackupServiceImpl(
       final Executor threadPoolTaskExecutor,
       final BackupPriorities backupPriorities,
       final BackupRepositoryProps backupProps,
-      final BackupRepository repository,
-      final DynamicIndicesProvider dynamicIndicesProvider) {
```

Comment:
You can probably keep this class like it was before and pass to the constructor a `DynamicIndicesProvider.noop` in the constructor

Reply:
That should make the test pass

Reply:
In the end I did something similar, indeed. Thanks